config/initializers/waste_exemptions_engine.rb

```diff
@@ -20,4 +20,7 @@
 
   # PDF config
   configuration.use_xvfb_for_wickedpdf = ENV["USE_XVFB_FOR_WICKEDPDF"] || "true"
+
+  # Last email cache config
+  configuration.use_xvfb_for_wickedpdf = ENV["USE_LAST_EMAIL_CACHE"] || "false"
```

Comment:
Think this should be `configuration.use_last_email_cache`.

Reply:
![facepalm](https://user-images.githubusercontent.com/1789650/55332936-356dfa80-548e-11e9-8385-71e6c432cbdc.gif)